package exceptionTest;

public class ExceptionTest4 {

	public static void main(String[] args) {

		//throw : 예외 발생시키기
		
		int num = 0;
		throw new NegativeArraySizeException
	}

}
